privacyidea/lib/importotp.py

```diff
@@ -683,5 +687,8 @@ def export_pskc(tokenobj_list, psk=None):
             number_of_exported_tokens += 1
         except Exception as e:
             log.warning(u"Failed to export the token {0!s}: {1!s}".format(serial, e))
+            tb = traceback.format_exc()
+            log.debug(tb)
+            raise(e)
```

Comment:
With this change, PSKC export fails if the export fails for any token. Just want to make sure that's intended? :-)

Reply:
No. Will do.

Reply:
RIght, it was for testing purposes and remained.

Reply:
done in d367ea32